src/css/base.forms.css

```diff
@@ -1,6 +1,75 @@
 /* ---------- BASE forms ---------- */
 
-/* ---------- select ---------- */
+/* input-search */
+.input[type="search"] {
```

Comment:
Question: Are there any other `input type="text"`s in the site? Wondering if this becomes more of a generic catch all for most "traditional" `input` types (text, date, email, phone, etc.) as well as search

Reply:
@adamduncan At this stage the only `input` is the search on the blog page.

If we can additional inputs in the future it'll be an easy CSS update.